TRACING.md

```diff
@@ -169,6 +169,10 @@ following:
      - The `jaeger-client-socat-redirector.service` systemd service
        (see the [Guest to Host communication using VSOCK](#guest-to-host-communication-using-vsock) section).
        This service should be placed at `/usr/lib/systemd/system/` of the rootfs.
+       It is necessary to perform the following command on the service:
+       ```
+       $ systemctl enable jaeger-client-socat-redirector.service
```

Comment:
Hi @GabyCT - can you make it clear these commands needs to be run from within the rootfs context. Something like:

```sh
$ sudo -E chroot "$rootfs" systemctl enable jaeger-client-socat-redirector.service
```